torch/_inductor/package/package.py

```diff
@@ -0,0 +1,214 @@
+import glob
+import io
+import json
+import os
+import re
+import shlex
+import subprocess
+import tempfile
+import zipfile
+from typing import Any, Callable, Dict, List, Union
+
+import torch
+import torch._inductor
+import torch.utils._pytree as pytree
+from torch._inductor import config, exc
+from torch._utils_internal import get_file_path_2
+from torch.export._tree_utils import reorder_kwargs
+
+
+ARCHIVE_VERSION = 0
+
+with open(
+    get_file_path_2(os.path.dirname(__file__), "pt2_archive_constants.json")
+) as f:
```

Comment:
Nit (if one to import `from pathlib import Path)
```suggestion
with (Path(__file__).parent / "pt2_archive_constants.json").open() as f:
```

Reply:
this seems to not work with CI ([example](https://github.com/pytorch/pytorch/actions/runs/9896316871/job/27339474170#step:20:766) :/